OracleDataIntegrator/dockerfiles/12.2.1.2.6/README.md

```diff
@@ -0,0 +1,143 @@
+ODI on Docker
```

Comment:
This file must be on folder `OracleDataIntegrator`, not on the subfolder `dockerfiles/12.2.1.2.6`